src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,67 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
 
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
-        System.out.println(movieService.get(fastAndFurious.getId()));
-        movieService.getAll().forEach(System.out::println);
+
+        //cascades
+        Movie fastAndStupid = new Movie("Fast and Stupid");
+        fastAndStupid.setDescription("An action comedy movie about romantic relationships");
+        Movie javaMovie = new Movie("MyJavaJourney");
+        javaMovie.setDescription("A thriller about setting out on a career in java");
+
+        CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+        CinemaHall extra = new CinemaHall(11,"11");
+        cinemaHallService.add(extra);
+        //cascades
+
+        System.out.println("------cinemaHallService:get1L: " + cinemaHallService.get(1L));
+        System.out.println("-------cinemaHallService:getAll(BEFORE CASCADES): "
+                + cinemaHallService.getAll());
+
+        MovieSession movieSession1 = new MovieSession();
```

Comment:
You are creating a `MovieSession` with a default constructor. Ensure that this is necessary and if not, consider removing it as it could be considered redundant functionality if not used properly.